Updated react-vis to 1.10.2

markov00 opened the pull request:
This PR updates `react-vis` lib to `1.10.2` and sticked to that version (see my comment on this https://github.com/uber/react-vis/issues/861)  that's includes our PR https://github.com/uber/react-vis/pull/857 to align XYPlot events handlers introducing some more event handlers we are using on EuiXYChart.
This PR get rid of the introduced  XYPlotExtended class used to include these handlers before the PR merge.

The package version in `package.json` is also updated since the PR was merged and available only from `1.10.2` version. 

The div around the XYPlot was also removed since it is not required anymore. All snapshots are updated after this removal.

chandlerprall commented on CHANGELOG.md:
```diff
@@ -1,6 +1,7 @@
 ## [`master`](https://github.com/elastic/eui/tree/master)
 
-No public interface changes since `1.2.1`.
+- No public interface changes since `1.2.1`.
```
remove the _no changes_ line

chandlerprall commented on src/components/xy_chart/axis/__snapshots__/horizontal_grid.test.js.snap:
```diff
@@ -5,80 +5,74 @@ exports[`EuiHorizontalGrid render the horizontal grid 1`] = `
-    aria-label="aria-label"
-    class="testClass1 testClass2"
-    data-test-subj="test subject string"
+    class="rv-xy-plot "
```
removing the wrapping `div` takes away support for `aria-label`, `class`, `data-test-subj`, and other custom attributes

cjcenizal commented:
@markov00 Sorry, I'm slammed so I'm going to have to remove myself from the review.

markov00 commented:
@chandlerprall I've updated the PR following your requested changes.
I've also fixed the version of react-vis to a fixed 1.10.2 because the next patch introduce some errors on tests and it's currently under discussion with uber guys.
